openscap_report/report_generators/html_templates/remedations.html

```diff
@@ -34,7 +34,18 @@
                     {% endif %}
                 </tbody>
             </table>
-            <pre>{{ remediation.fix | e }}</pre>
+            <div class="pf-c-clipboard-copy pf-m-inline pf-m-block remediation-clipboard-padding">
+                <span class="pf-c-clipboard-copy__actions">
+                  <span class="pf-c-clipboard-copy__actions-item">
+                    <button id="copy" class="pf-c-button pf-m-plain" type="button" aria-label="Copy to clipboard">
+                      <i class="fas fa-copy" aria-hidden="true"></i> Copy remediation to clipboard!
```

Comment:
Don't yell at the user or get over-exited. Simple `[icon] Copy Remediation` should be enough.